tools/devtool

```diff
@@ -596,26 +602,31 @@ cmd_prepare_release() {
     files_to_change=("$swagger"                         \
                      "$FC_ROOT_DIR/Cargo.toml"          \
                      "$FC_ROOT_DIR/jailer/Cargo.toml")
+    say "Updating source files:"
     for file in "${files_to_change[@]}"; do
-        sed -i "s/$curr_ver/$version/g" "$file"
+        say "- $file"
+        # Dirty hack to make this work on both macOS/BSD and Linux.
+        sed -i="" "s/$curr_ver/$version/g" "$file"
+        rm -f "${file}="
```

Comment:
Is this a weird `sed` artefact on non-gnu? Also, would it help to be consistent with the variables, i.e. enclose all of them in `{}`?

Reply:
Yeah, on macOS, the backup file suffix following `-i` is mandatory, unless empty. In this case, it will create a backup file with a `=` suffix, that we need to remove.

I think readability trumps consistency on the curly-bracket-or-no issue. I.e. bash syntax is ugly and confusing as hell, so anything we can do to simplify it, I find welcome.

Reply:
Yeah, on macOS, the backup file suffix following `-i` is mandatory, unless empty. In this case, it will create a backup file with a `=` suffix, that we need to remove.

I think readability trumps consistency on the curly-bracket-or-no issue. I.e. bash syntax is ugly and confusing as hell, so anything we can do to simplify it, I find welcome.